### PR TITLE
Contextual tabs again

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by SylvaD-MS** [comment]
The new verbose syntax is not correct (the groupeName is **CustomGroup111**) and doesn't match the previous syntax for button enabled (true|false) property https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/disable-add-in-commands?branch=pr-en-us-2415 
      "groups": [
        {
          "id": "CustomGroup111",

Button ids must be unique therefore you don't need to specify the group.

**Comment by Rick-Kirkham** [comment]
> The new verbose syntax is not correct (the groupeName is **CustomGroup111**) and doesn't match the previous syntax for button enabled (true|false) property https://review.docs.microsoft.com/en-us/office/dev/add-ins/design/disable-add-in-commands?branch=pr-en-us-2415
> "groups": [
> {
> "id": "CustomGroup111",
> 
> Button ids must be unique therefore you don't need to specify the group.

This is moot now because we've decided not to advertise the more concise syntax (even though it will still work).

File: docs/design/disable-add-in-commands.md [hunk 1]
```diff
@@ -97,7 +104,8 @@ We also provide several interfaces (types) to make it easier to construct the **
 ```typescript
 const enableButton = async () => {
     const button: Control = {id: "MyButton", enabled: true};
-    const parentTab: Tab = {id: "OfficeAddinTab1", controls: [button]};
+    const parentGroup: Group = {id: "CustomGroup111", controls: [button]}
```
Comment (alison-mk) [inline docs/design/disable-add-in-commands.md:107]:
```suggestion
    const parentGroup: Group = {id: "CustomGroup111", controls: [button]};
```

File: docs/design/disable-add-in-commands.md [hunk 2]
```diff
@@ -71,24 +71,31 @@ By default, any Add-in Command is enabled when the Office application launches.
 
 The essential steps to changing the enabled status of an Add-in Command are:
 
-1. Create a [RibbonUpdaterData](/javascript/api/office/office.ribbonupdaterdata) object that (1) specifies the command, and its parent tab, by their IDs as specified in the manifest; and (2) specifies the enabled or disabled state of the command.
+1. Create a [RibbonUpdaterData](/javascript/api/office/office.ribbonupdaterdata) object that (1) specifies the command, and its parent group and tab, by their IDs as specified in the manifest; and (2) specifies the enabled or disabled state of the command.
```
Comment (alison-mk) [inline docs/design/disable-add-in-commands.md:74]: Would it still be accurate if one of the instances of 'specify' is changed? Seeing 'specify' so many times in a row makes understanding the sentence difficult for me. 
```suggestion
1. Create a [RibbonUpdaterData](/javascript/api/office/office.ribbonupdaterdata) object that (1) specifies the command, and its parent group and tab, by their IDs as declared in the manifest; and (2) specifies the enabled or disabled state of the command.
```

**Review by alison-mk — APPROVED** [review]
Approved with two suggestions.

**Pull request merged**